contrib/scalar/scalar.c

```diff
@@ -770,6 +770,25 @@ int cmd_main(int argc, const char **argv)
 	struct strbuf scalar_usage = STRBUF_INIT;
 	int i;
 
+	while (argc > 1 && *argv[1] == '-') {
```

Comment:
nit: the commit message uses back-ticks `git` when the typical Git style is to use single ticks ('git').

Reply:
Hmm. It is _my_ common commit message style to use backticks where code, or code-like text, is involved.

Reply:
Ok. Use your style.